app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# https://github.com/plataformatec/devise/wiki/OmniAuth:-Overview
+#
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```

Comment:
@rschenk @whereismyjetpack None of this class is showing as tested. Do you have unit tests to cover the cases?

Reply:
We didn't write this code, it came straight from Devise and was hard af to test, so I didn't. But I can try to figure it out!